draft-ietf-quic-recovery.md

```diff
@@ -410,11 +417,11 @@ endpoint:
   min_rtt.  This limits the underestimation that a misreporting peer can cause
   to the smoothed_rtt.
 
-On the first RTT sample in a connection, the smoothed_rtt is set to the
-latest_rtt.
+On the first RTT sample in a congestion control context, the smoothed_rtt is
```

Comment:
RTT context

Reply:
path?

Reply:
Elsewhere don’t we discuss resetting a congestion control context? It seems like that often lines up with a path, but they don’t have to be the same thing. It seems like you’re saying that RTT isn’t part of congestion control (which sounds totally valid but we then need some better way to word that)?